src/Illuminate/Queue/Console/PruneBatchesCommand.php

```diff
@@ -26,17 +25,14 @@ class PruneBatchesCommand extends Command
     /**
      * Execute the console command.
      *
+     * @param  PrunableBatchRepository $repository
      * @return void
      */
-    public function handle()
+    public function handle(PrunableBatchRepository $repository)
```

Comment:
As far as I can tell, This interface is not in the container and won’t be resolved and the command will break. 

The point of the original check was that some future repository may not be prunable  so it should still use the BatchRepository interface to get the repository and I think the it check is still required.